From c800aeea1ba725dce5badc0ab3af788c3861014a Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 06 May 2019 20:24:33 +0300
Subject: [PATCH] [SW-0] asic: move include to common.h

The local includes usually should come after the global ones as those
are more generic.
Move the local include to common.h after the global ones.

Change-Id: Ib004be2c1f343b6e6d33aa0077817030ca132445
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/asic.c b/gaudi/asic.c
index c540cf3..922e4a6 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -5,14 +5,14 @@
  * SPDX-License-Identifier:     GPL-2.0+
  */
 
-#include "common.h"
-
 #include <psoc_global_conf_regs.h>
 
 #include <preboot.h>
 #include <pll.h>
 #include <uart.h>
 
+#include "common.h"
+
 static struct hl_pll_cfg gaudi_pll_cfg[] = {
 	{
 		.name = "DMA_W", .base = mmDMA_W_PLL_BASE,
diff --git a/goya/asic.c b/goya/asic.c
index d1c0add..174e586 100644
--- a/goya/asic.c
+++ b/goya/asic.c
@@ -5,14 +5,14 @@
  * SPDX-License-Identifier:     GPL-2.0+
  */
 
-#include "common.h"
-
 #include <psoc_global_conf_regs.h>
 
 #include <preboot.h>
 #include <pll.h>
 #include <uart.h>
 
+#include "common.h"
+
 static struct hl_pll_cfg goya_pll_cfg[] = {
 	{
 		.name = "IC", .base = mmIC_PLL_BASE,
